images/win/scripts/Installers/Install-PowerShellModules.ps1

```diff
@@ -0,0 +1,39 @@
+################################################################################
+##  File:  Install-PowerShellModules.ps1
+##  Desc:  Install PowerShell Modules.
+################################################################################

```

Comment:
Do we really need this header? I see that some script contains it but it doesn't look useful because just duplicate script name